digiroad2-api-viite/src/main/scala/fi/liikennevirasto/digiroad2/IntegrationApi.scala

```diff
@@ -285,15 +285,13 @@ class IntegrationApi(val roadAddressService: RoadAddressService, val roadNameSer
       "road_part" -> nodePoint.roadPartNumber,
       "track" -> nodePoint.track.value,
       "distance" -> nodePoint.addrM,
-      "start_date" -> nodePoint.startDate.toString,
-      "end_date" -> (if(nodePoint.endDate.isDefined) nodePoint.endDate.get.toString else null),
       "user" -> nodePoint.createdBy
     )
   }
 
   def junctionToApi(junction: (Junction, Seq[JunctionPoint])): Map[String, Any] = {
     Map(
-      "junction_number" -> junction._1.junctionNumber,
+      "junction_number" -> (if (junction._1.junctionNumber.isDefined) junction._1.junctionNumber.get else null),
```

Comment:
since junctionNumber is now an Option, you can should be able to do "junction._1.junctionNumber.getOrElse(null)"

Reply:
> since junctionNumber is now an Option, you can should be able to do "junction._1.junctionNumber.getOrElse(null)"

Replaced by: junction._1.junctionNumber.orNull